test/pcmdi/installationTest/tas_2.5x2.5_regrid2_linear_metrics.json

```diff
@@ -389,6 +389,6 @@
             "Name": "Custom Maximum"
         }
     },
-    "metrics_git_sha1": "40e056c9ec4b9ec96eb5c3a93ad89fcd427429f5",
-    "uvcdat_version": "2.1.0-682-g5f296b7"
+    "metrics_git_sha1": "3100dca207554bffd8cc9c56b7d3887c47a3f14b",
+    "uvcdat_version": "uvcdat-2.2.0-320-g80914b7"
```

Comment:
My question was more about the numbers in the `json` file, as we know roundoff and masking can cause these to move around a bit.. `2.2.0` vs `2.4.0` might give subtly different numbers that ARE being tested..